ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/LedgerTransaction.kt

```diff
@@ -0,0 +1,12 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.crypto.SecureHash
+import java.security.PublicKey
+
+@DoNotImplement
+interface LedgerTransaction {
+    val id: SecureHash
+    val wireTransaction: WireTransaction // TODO(to be removed if wireTx gets private)
```

Comment:
I don't think we need this. We don't convert LedgerTransaction back to WireTransaction

Reply:
Thanks. I have moved it to private/runtime-os.